apm-collector/apm-collector-boot/src/main/resources/application.yml

```diff
@@ -1,84 +1,90 @@
-
-    #Set your own token to active auth
-    #authentication: xxxxxx
-agent_jetty:
-  jetty:
-    host: localhost
-    port: 12800
-    contextPath: /
-analysis_register:
-  default:
-analysis_jvm:
-  default:
-analysis_segment_parser:
-  default:
-    bufferFilePath: ../buffer/
-    bufferOffsetMaxFileSize: 10M
-    bufferSegmentMaxFileSize: 500M
-ui:
-  jetty:
-    host: localhost
-    port: 12800
-    contextPath: /
-storage:
-  elasticsearch:
-    clusterName: CollectorDBCluster
-    clusterTransportSniffer: true
-    clusterNodes: localhost:9300
-    indexShardsNumber: 2
-    indexReplicasNumber: 0
-    highPerformanceMode: true
-    ttl: 7
-#storage:
-#  h2:
-#    url: jdbc:h2:~/memorydb
-#    userName: sa
-configuration:
-  default:
-#     namespace: xxxxx
-    applicationApdexThreshold: 2000
-    serviceErrorRateThreshold: 10.00
-    serviceAverageResponseTimeThreshold: 2000
-    instanceErrorRateThreshold: 10.00
-    instanceAverageResponseTimeThreshold: 2000
-    applicationErrorRateThreshold: 10.00
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#cluster:
+#  zookeeper:
+#    hostPort: localhost:2181
+#    sessionTimeout: 100000
+naming:
+  jetty:
+    host: localhost
+    port: 10800
+    contextPath: /
+cache:
+#  guava:
+  caffeine:
+remote:
+  gRPC:
+    host: localhost
+    port: 11800
+agent_gRPC:
+  gRPC:
+    host: localhost
+    port: 11800
+    #Set these two setting to open ssl
+    #sslCertChainFile: $path
+    #sslPrivateKeyFile: $path
+
+    #Set your own token to active auth
+    #authentication: xxxxxx
+agent_jetty:
+  jetty:
+    host: localhost
+    port: 12800
+    contextPath: /
+analysis_register:
+  default:
+analysis_jvm:
+  default:
+analysis_segment_parser:
+  default:
+    bufferFilePath: ../buffer/
+    bufferOffsetMaxFileSize: 10M
+    bufferSegmentMaxFileSize: 500M
+ui:
+  jetty:
+    host: localhost
+    port: 12800
+    contextPath: /
+storage:
+  elasticsearch:
+    clusterName: CollectorDBCluster
+    clusterTransportSniffer: true
+    clusterNodes: localhost:9300
+    indexShardsNumber: 2
+    indexReplicasNumber: 0
+    highPerformanceMode: true
+    ttl: 7
+#storage:
+#  h2:
+#    url: jdbc:h2:~/memorydb
+#    userName: sa
+#storage:
+#  shardingjdbc:

```

Comment:
ShardingJDBC is an incubating feature, no use cases yet. And none of the contributors are committer yet. So, by that, please don't put Sharding JDBC in default application.yml. Even it is under comments.